common/djangoapps/student/models.py

```diff
@@ -2744,3 +2744,38 @@ class Meta(object):
         db_table = "auth_accountrecovery"
 
     objects = AccountRecoveryManager()
+
+
+class AllCoursesEnrollmentBanned(models.Model):
```

Comment:
Can we rename this model `AllCoursesEnrollmentBanned` doesn't sound's good? `EnrollmentBanned` is enough I guess.